selectors-4/Overview.bs

```diff
@@ -585,16 +586,16 @@ Structure and Terminology</h3>
 	the <a>child combinator</a> (U+003E, <code>></code>),
 	the <a>next-sibling combinator</a> (U+002B, <code>+</code>),
 	and the <a>subsequent-sibling combinator</a> (U+007E, <code>~</code>).
-	Two given elements are said to <a>match</a> a <a>combinator</a>
+	Two given elements are said to <a>match</a> a <a for=selector>combinator</a>
 	when the condition of relationship between these elements is true.
 
 	A <dfn id="complex" export>complex selector</dfn> is
 	a sequence of one or more <a>compound selectors</a>
 	and/or [=pseudo-compound selectors=],
-	with [=compound selectors=] separated by <a>combinators</a>.
+	with [=compound selectors=] separated by <a for=selector>combinators</a>.
 	It represents a set of simultaneous conditions
 	on a set of elements in the particular relationships
-	described by its <a>combinators</a>.
+	described by its <a for=selector>combinators</a>.
```

Comment:
This edit fixes the warning, but for some reason it's still not link-ifying, and I'm not sure why.

Reply:
Oh, it works just fine, it's just that, in order to reduce visual clutter in heavily linked text, Bikeshed suppresses instances of a given autolink past the first in a given paragraph. (We tried to just visually reduce the contrast, but that was nixed by a11y people, so instead of "this link exists, but it's intentionally a little hard to distinguish from surrounding text" we just have "this link doesn't exist at all" now. :( )